public/stylesheets/app/_modal_popup.scss

```diff
@@ -0,0 +1,60 @@
+body{
+  width: 100%;
+}
+
+.modal-container {
+  display: none; /* Hidden by default */
+  position: fixed; /* Stay in place */
+  z-index: 1; /* Sit on top */
+  left: 0;
+  top: 0;
+  width: 100%; /* Full width */
+  height: 100%; /* Full height */
+  overflow: auto; /* Enable scroll if needed */
+  background-color: rgb(0,0,0); /* Fallback color */
+  background-color: rgba(0,0,0,0.4); /* Black w/ opacity */
+}
+
+.modal-close {
+  float: right!important;
+  outline: none;
+  appearance: none;
+  color: black;
+  background: none!important;
+  border: none!important;
+  font-weight: bold;
+  cursor: pointer;
+}
+
+/* Modal Content/Box */
+.modal-content {
+  background-color: #fefefe;
+  margin: 15% auto;
+  padding: 20px;
+  border: 2px solid #000000!important;
+  width: 35%!important;
```

Comment:
seeing a lot of `!imortant` flags here, could you not have targeted with css selectors to be more specific instead?

Reply:
Not good at this css stuff. Happy for you to contribute on this.